packages/runtime/test/src/interpreter/interpreter.ts

```diff
@@ -0,0 +1,126 @@
+import { bobAcc } from "@algo-builder/algob/test/mocks/account";
+import { decodeAddress, getApplicationAddress } from "algosdk";
+import { assert } from "chai";
+
+import { AccountStore, getProgram, Interpreter, Runtime } from "../../../src";
+import RUNTIME_ERRORS from "../../../src/errors/errors-list";
+import { ALGORAND_ACCOUNT_MIN_BALANCE } from "../../../src/lib/constants";
+import { Stack } from "../../../src/lib/stack";
+import { AccountAddress, AccountStoreI, ExecutionMode, StackElem } from "../../../src/types";
+import { useFixture } from "../../helpers/integration";
+import { expectRuntimeError } from "../../helpers/runtime-errors";
+import { elonMuskAccount, johnAccount } from "../../mocks/account";
+import { accInfo } from "../../mocks/stateful";
+import { elonAddr, TXN_OBJ } from "../../mocks/txn";
+
+export function setDummyAccInfo(acc: AccountStoreI): void {
+	acc.appsLocalState = accInfo[0].appsLocalState;
+	acc.appsTotalSchema = accInfo[0].appsTotalSchema;
+	acc.createdApps = accInfo[0].createdApps;
+}
+
+describe("Interpreter", function () {
+	let interpreter: Interpreter;
+	let elonAcc: AccountStoreI;
```

Comment:
take a look at the `defaultAccounts` in runtime. We use them in other tests

Reply:
I use `defaultAccounts` which has already set up in the inner-transactions file.
So is there things get wrong here?